package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@celo/celo-composer",
-  "version": "0.0.2-alpha",
+  "version": "0.0.3",
```

Comment:
We can make the version `1.0.1` as the current live version on npm is `1.0.0`.

Reply:
done.